lib/App/Asciio/Actions/Unsorted.pm

```diff
@@ -91,6 +92,8 @@ my ($self, $direction) = @_ ;
 
 my ($family, $size) = $self->get_font() ;
 
+my ($ori_character_width, $ori_character_height) = $self->get_character_size() ;
+
 $self->set_font($family, $size + $direction) ;
```

Comment:
set_font is what is making  the character size change, a comment here would be welcome

# change font size which changes the character size
$self->set_font($family, $size + $direction) ;

adding :
my ($new_character_width, $new_character_height) = $self->get_character_size() ;

just after the font setting makes things even more obvious


and since the font change is only happening in the GUI, why is the code above not in the "if($self->{UI} eq 'GUI')" block?